pkg/services/ngalert/models/alert_rule.go

```diff
@@ -169,6 +176,11 @@ func (alertRule *AlertRule) GetKey() AlertRuleKey {
 	return AlertRuleKey{OrgID: alertRule.OrgID, UID: alertRule.UID}
```

Comment:
If you do the embedding, you can forward this func to the embedded struct's func